### ISSUE TITLE
Add Masked Prediction as Training regime

### ISSUE THREAD

**Issue by RaphaelKreft** [RaphaelKreft/DeepLearningProject#5]

**Comment by RaphaelKreft** [comment]
Steps for completion:

- [ ] Add Masked autoencoder model 
- [ ] Integrate Masked Autoencoder Model into the repo (so it works with training scripts)
    - [ ] Check if dataloader gives correct format
    - [ ] Check what the settings for equence length and batch size really mean
    - [ ] Make the Masked Autoencoder configurable (architecture, freeze, ...)

- [ ] Add training script specific to MP and add train config
